contracts/external/actions/TreasuryAction.sol

```diff
@@ -124,7 +124,7 @@ contract TreasuryAction is StorageLayoutV2, ActionGuards, NotionalTreasury {
 
         require(config.holding == holding);
         require(config.targetUtilization < 100);
-        require(100 <= config.externalWithdrawThreshold);
+        require(100 < config.externalWithdrawThreshold);
```

Comment:
@T-Woodward should this always be greater than 100?

Reply:
Not necessarily. Again, could be something like sDAI where there is no redemption/illiquidity risk. In that case you wouldn't need a withdrawal buffer theoretically